Backends/common/fonts.lisp

```diff
@@ -187,10 +187,13 @@ of letters specified in a separate kerning-table."))
 
 (declaim (inline climb:normalize-font-size))
 (defun climb:normalize-font-size (size)
-  (if (numberp size)
-      (round (max size 2))
-      (or (getf +font-sizes+ size size)
-          (getf +font-sizes+ :normal))))
+  (cond ((numberp size)
+         (round (max size 2)))
+        ((null size)
+         (getf +font-sizes+ (text-style-size *default-text-style*)))
+        (T
+         (or (getf +font-sizes+ size size)
+             (error "~s is not a valid text style size!" size)))))
```

Comment:
How does this work? In this leg of the `cond`, `size` is not null, thus `(getf +font-sizes+ size size)` should not return null.

Reply:
you are right, fixed in the force-pushed version.

Reply:
`getf` defaults to `nil` so the third argument could be omitted. Not super important.

Reply:
I want to put emphasis on returning NIL if not found